### PR TITLE
fixed the noScholarshipsYet bug.

### PR DESCRIPTION
**Pull request by Jobhdez**
## Motivation and Context

I fixed the bug and addressed Josue's new review comments. Now, I think, everything is working as expected.

## How Has This Been Tested?

the message "No Scholarships Added" does not appear on the user's homepage when the user has added scholarships.

moreover, the bug that made the grid container, with the inbox icon as an item, appear after reloading the page no longer appears. after I made my changes, I logged in and reloaded the page a couple of times and nothing appeared. 

I also centered the text and link(ie items) inside the grid container. I also had misspelled a keyword so fixed that too. And I made a few other small changes. 

## Types of changes
Fixed a bug.

- [ x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] User visible change (users will notice UI or functional changes)

## Previewing Changes

1. Go to https://s4us-pr-NUMBER.onrender.com.
2.

## Checklist:

- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: src/pages/UserHome.jsx [hunk 1]
```diff
@@ -117,7 +117,7 @@ export default function UserHome() {
           component="h2"
           gutterBottom
           className={classes.noScholarshipsAdded}>
-          No Scholarships Added Yet
+          <div>{scholarships.length === 0 && 'No Scholarships added.'}</div>
```
Comment (josueetcom) [inline src/pages/UserHome.jsx:120]: This isn't the only piece of the code that needs to be conditionally rendered. Look at Figma design and compare what the page looks like [with scholarships](https://www.figma.com/file/nwrW3UebcvSWyQZGb5ixNr/DreamScholar?node-id=562%3A1691) and [without scholarships](https://www.figma.com/file/nwrW3UebcvSWyQZGb5ixNr/DreamScholar?node-id=562%3A1809)

**Review by josueetcom — DISMISSED** [review]

File: src/pages/UserHome.jsx [hunk 2]
```diff
@@ -109,35 +107,40 @@ export default function UserHome() {
...
             <ScholarshipList scholarships={scholarships} />
-            <Button
-              className={classes.loadMoreButton}
-              color="primary"
-              onclick={() => {
-                alert('clicked');
-              }}>
-              Load More
-            </Button>
+            {scholarships.length !== 0 && (
```
Comment (josueetcom) [inline src/pages/UserHome.jsx:134]: This is not really the right condition for this but I guess it works for now.

File: src/pages/UserHome.jsx [hunk 3]
```diff
@@ -109,35 +107,40 @@ export default function UserHome() {
             Add Scholarship
           </Button>
         </Grid>
+        {scholarships.length === 0 && (
```
Comment (josueetcom) [inline src/pages/UserHome.jsx:110]: This makes this show up when scholarships are loading. 

https://user-images.githubusercontent.com/8023233/110750375-39b4fd80-81f7-11eb-89ea-088a711dac04.mp4

If there's an error or scholarships are loading this shouldn't show up. Look at how `ScholarshipList` is conditionally loaded below for an example on how to do this. In fact, this should probably go there

Comment (josueetcom) [reply src/pages/UserHome.jsx:110]: What you want to do is, just like the actual scholarships themselves below, only show the whole "no scholarships added yet" thing when there are no errors and nothing is loading.

File: src/pages/UserHome.jsx [hunk 4]
```diff
@@ -109,35 +107,40 @@ export default function UserHome() {
...
+          <Grid container className={classes.bannerRoot}>
+            <Grid item xs={3}>
+              <InboxIcon style={{ fontsize: 600 }} />
```
Comment (josueetcom) [inline src/pages/UserHome.jsx:113]: The [Icon property](https://material-ui.com/api/icon/#props) is called `fontSize` not `fontsize`.

File: src/pages/UserHome.jsx [hunk 5]
```diff
@@ -109,35 +107,40 @@ export default function UserHome() {
...
+                No Scholarships Added Yet{' '}
+              </Typography>
+              <Typography
+                variant="contained"
+                color="primary"
+                component={Link}
+                to="/scholarships/new">
+                Add Scholarship
+              </Typography>
+            </Grid>
```
Comment (josueetcom) [inline src/pages/UserHome.jsx:126]: These should be vertically centered. Take a look at the [Grid](https://material-ui.com/components/grid/) docs to learn how to do this. The [Interactive](https://material-ui.com/components/grid/#interactive) section might be helpful.

File: src/pages/UserHome.jsx [hunk 6]
```diff
@@ -109,35 +107,40 @@ export default function UserHome() {
...
           </Button>
         </Grid>
+        {scholarships.length === 0 && (
+          <Grid container className={classes.bannerRoot}>
+            <Grid item xs={3}>
+              <InboxIcon style={{ fontsize: 600 }} />
+            </Grid>
+            <Grid item>
+              <Typography variant="h5" gutterButtom>
+                No Scholarships Added Yet{' '}
```
Comment (josueetcom) [inline src/pages/UserHome.jsx:117]: No need for this extraneous space:
```suggestion
                No Scholarships Added Yet
```

File: src/pages/UserHome.jsx [hunk 7]
```diff
@@ -109,35 +107,40 @@ export default function UserHome() {
...
+              </Typography>
+              <Typography
+                variant="contained"
+                color="primary"
+                component={Link}
+                to="/scholarships/new">
+                Add Scholarship
+              </Typography>
```
Comment (josueetcom) [inline src/pages/UserHome.jsx:125]: The Typography/MuiLink structure you had before was fine. Why the change?

File: src/pages/UserHome.jsx [hunk 8]
```diff
@@ -109,35 +107,40 @@ export default function UserHome() {
...
+          </Grid>
+        )}
       </Grid>
-      <Box className={classes.noScholarshipsBox}>
-        <InboxIcon style={{ fontSize: 200 }} />
-        <Typography
-          variant="h5"
-          component="h2"
-          gutterBottom
-          className={classes.noScholarshipsAdded}>
```
Comment (josueetcom) [inline src/pages/UserHome.jsx:119]: Please remove the CSS classes no longer used from above.

**Review by josueetcom — CHANGES_REQUESTED** [review]

File: src/pages/UserHome.jsx (hunk 7 above)
Comment (Jobhdez) [reply src/pages/UserHome.jsx]: I left it like that because it didnt seem you were being strict about it and I thought it was not really necessary to change, but I addressed all of your other comments.

Comment (josueetcom) [reply src/pages/UserHome.jsx]: Oh, well actually the _original_ code used the Typography/MuiLink structure. That's why I was asking why it was changed. It can be left the way you changed it as the styling for the page needs to be completed later anyways.

File: src/pages/UserHome.jsx [hunk 9]
```diff
@@ -110,34 +102,43 @@ export default function UserHome() {
...
+            {scholarships.length !== 0 && (
+              <Button
+                className={classes.loadMoreButton}
+                color="primary"
+                onclick={() => {
+                  alert('clicked');
+                }}>
+                Load More
+              </Button>
+            )}
```
Comment (josueetcom) [inline src/pages/UserHome.jsx:141]: I'm not sure how you've read into [conditional rendering](https://reactjs.org/docs/conditional-rendering.html) but there's a section on [inlining if-else](https://reactjs.org/docs/conditional-rendering.html#inline-if-else-with-conditional-operator) within the JSX. We generally use that where possible. But, since this needs to be updated anyways when the **Load More** button works, making this change is optional.

**Review by josueetcom — APPROVED** [review]
I think this is good to go for fixing the core issue.
